aiohttp/http_websocket.py

```diff
@@ -596,7 +598,7 @@ def __init__(
         self.protocol = protocol
         self.transport = transport
         self.use_mask = use_mask
-        self.randrange = random.randrange
+        self.get_random_bits = partial(random.getrandbits, 32)
```

Comment:
Masking key is a 32-bit value https://datatracker.ietf.org/doc/html/rfc6455#section-5.3